### ISSUE TITLE
Status blog

### ISSUE THREAD

**Issue by rdebeasi** [openpracticelibrary/openpracticelibrary#258]
Let's share status updates on the practice library itself. Currently we're doing this via a mailing list, but there have been requests to publish this information so that it's a.) shareable and b.) available to the public.

Ideas:
- Hugo blog posts in openpracticelibrary.com
- Public-facing email list (maybe generated from blog posts?)

**Comment by tdbeattie** [comment]
+1 to above.

Further idea:
- Twitter account for the openpracticelibrary

**Comment by rdebeasi** [comment]
Question: is someone available to maintain and update this blog regularly? I wonder if it would be useful to link to our [recent merges to staging](https://github.com/openpracticelibrary/openpracticelibrary/commits/staging) and/or pull that information into a page or a twitter bot. I worry that if we create a blog and then don't update it regularly, that might do more harm than good.

We've also moved our conversations to GitHub issues and gitter, so the "move it out of email" use case is (I think!) handled.

**Comment by rdebeasi** [comment]
There are also these resources:

- [staging branch RSS feed](https://github.com/openpracticelibrary/openpracticelibrary/commits/staging.atom)
- [master branch RSS feed](https://github.com/openpracticelibrary/openpracticelibrary/commits/master.atom)
- [GitHub pulse](https://github.com/openpracticelibrary/openpracticelibrary/pulse)

**Comment by mtakane** [comment]
@rdebeasi good question. I'm not certain I can regularly maintain it, I do think if we get ahead of things and have something like what you suggest for publishing recent merges to staging (for moderators to check/merge into master), and then a public sharing on some release notes page it would be a good start.

**Comment by rdebeasi** [comment]
In the future, we could use GitHub releases to deploy to production, rather than merging from staging to master. GitHub releases provides an easy way to write up release notes for each release. This would also get us more in line with what we'd recommend when working with a container platform: deploy each environment off of the master branch, and use tags to control the version of the code in each environment.

Problems:
- Netlify [doesn't seem to support deploying via tags](https://github.com/mochajs/mocha/issues/3206#issuecomment-385365665)
- Netlify CMS [uploads images to the "main" branch](https://github.com/netlify/netlify-cms/issues/1344), not to PR branches. We made the "main" branch `staging` to address this limitation, but the caveat is that someone does need to inspect these changes before they go to prod.

Possible solutions:
- Use [Netlify Actions])(https://github.com/netlify/actions) to deploy (once this feature is ready)
- Fix the Netlify CMS issue upstream
